tests/social_smells_tests.R

```diff
@@ -0,0 +1,75 @@
+library("testthat")
+library("kaiaulu")
+library("data.table")
+library("stringi")
+source("~/kaiaulu/R/interval.R")
```

Comment:
These lines are not needed when executing the test via RStudio. Try to add a file to Kaiaulu, and then use Build Tab -> More -> Test Package. 

The file should also be renamed to test-interval.R since that's the file being tested here.